e2e/errors.go

```diff
@@ -43,3 +43,20 @@ func isRetryableAPIError(err error) bool {
 
 	return false
 }
+
+// isAlreadyExistsCLIError checks for already exists error from kubectl CLI.
+func isAlreadyExistsCLIError(err error) bool {
+	if err == nil {
+		return false
+	}
+	// if multiple resources already exists. each error is separated by newline
+	errStr := strings.Split(err.Error(), "/n")
```

Comment:
use `\n`?

Reply:
:facepalm: yes because of this one I was not able to figure out the range problem. will fix it now